### PR TITLE
SUIT-15442 - TypeError on displaying unused leaves warning

### PR DESCRIPTION
**Pull request by AntonSBilan**

### CODE REVIEW

File: test/chains/elementChain.test.js [hunk 1]
```diff
@@ -318,6 +318,8 @@ describe('Element chain', () => {
 			element('el-api-id').setText('text string').toString(),
 			'Setting text "text string" for "el-api-id"'
 		);
+		assert.strictEqual(elementAssert({css: 'any'}).toString(), '');
```
Comment (JeKachan) [inline test/chains/elementChain.test.js:321]: I think, should be returned something similar like for not asserted command:
```javascript
suitest.element({ css: 'body' }).toString(); // Getting properties of "{"css":"body"}"
```

Comment (AntonSBilan) [reply test/chains/elementChain.test.js]: Did it similar to application chain (throw invalid arguments error)

File: lib/utils/makeChain.js [hunk 2]
```diff
@@ -14,7 +15,11 @@ const makeChain = (getComposers, data) => {
 		return chain;
 	});
 
-	return makeNewChain(data);
+	const chain = makeNewChain(data);
+
+	registerLeaf(chain, data.stack);
```
Comment (JeKachan) [inline lib/utils/makeChain.js:20]: If I understand correct it should cover: `suitest.element(selector)` ?

File: typeDefinition/modifiers.d.ts [hunk 3]
```diff
@@ -4,6 +4,7 @@ export interface Thenable <R> {
 }
 
 export interface AbstractChain {
+	// May throw SuitestError if chain is malformed
```
Comment (JeKachan) [inline typeDefinition/modifiers.d.ts:7]: Could you please replace it by jsdoc @throws directive?

**Pull request merged**